### ISSUE TITLE
Exclusion tables: all rows must have an exclusion reason

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo-ingest#85]
Please make sure that all exclusion table rows have exclusion reasons documented.

Once this is done, we should add a check in your code @joeflack4 that makes the extensional list generation fail if an exclusion reason is not provided.

**Comment by joeflack4** [comment]
OK, that is an easy thing for me to add. I'll await when this is ready for me to add that.

**Comment by joeflack4** [comment]
Yep, those are the ones that are missing exclusion reasons from what I can see in `mondo-ingest` `src/ontology/config/`.

**Comment by sabrinatoro** [comment]
OK, I'll get on it!
